ui/src/overview.tsx

```diff
@@ -494,51 +465,36 @@ function DemographicCharts({ cohort }: DemographicChartsProps) {
   const underlay = useUnderlay();
   const source = useSource();
 
-  const api = useContext(EntityCountsApiContext);
-
   const generatePropertyString = (
     property: ChartConfigProperty,
-    entityCountStruct: tanagra.EntityCountStruct
+    filterCountValue: FilterCountValue
   ) => {
     let propertyString = "";
     // TODO(neelismail): Remove property key check once API supports age.
-    const entityCountPropertyValue =
-      entityCountStruct.definition?.[
-        property.key === "age" ? "year_of_birth" : property.key
-      ];
-
-    if (entityCountPropertyValue) {
-      let value =
-        entityCountPropertyValue.int64Val ??
-        entityCountPropertyValue.stringVal ??
-        entityCountPropertyValue.boolVal;
-
-      // TODO(neelismail): Remove age handling once the API supports them.
-      if (
-        isValid(value) &&
-        property.key === "age" &&
-        typeof value === "number"
-      ) {
-        value = new Date().getFullYear() - value;
-      }
+    let value =
+      filterCountValue[property.key === "age" ? "year_of_birth" : property.key];
```

Comment:
Right now, the breakdown chart attributes are hard-coded in the UI. Should we make these part of the entity underlay definition (e.g. flag attributes as "forBreakdown")?

Reply:
The only thing that's hardcoded is the `age => year_at_birth` conversion. The remaining attributes and the fact that `age` is one of them are already coming from the underlay.